app/src/main/java/com/epmedu/animeal/MainActivity.kt

```diff
@@ -23,6 +23,11 @@ class MainActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
+        Thread.UncaughtExceptionHandler { t, e ->
```

Comment:
Why we need this?

Reply:
Sorry, it is only for debug on my device.

Reply:
Sorry, it is only for debug on my device.